### PR TITLE
Make some validation changes

### PR DESCRIPTION
**Pull request by jennydely**
Add some new validation requirements

#34

### CODE REVIEW

File: src/Packlist/PacklistForm.js [hunk 1]
```diff
@@ -98,24 +99,18 @@ export default function PacklistForm({ onPacklistSave }) {
...
-          {(errors.item?.type === 'validate' ||
-            errors.item?.type === 'maxLength') && (
+
+          {itemError && (
             <ErrorMessage>
-              The Item can reach a maximum of 20 characters!
+              'This item field requires at least 3 characters and can reach a
+              maximum of 20 characters!'
             </ErrorMessage>
           )}
```
Comment (carolinwagner) [inline src/Packlist/PacklistForm.js:108]: Out of curiosity: why did you move the validation from react-hook-form to your own state?

Comment (jennydely) [reply src/Packlist/PacklistForm.js:108]: because react hook is not working here
